db/pg/index.js

```diff
@@ -19,7 +19,7 @@ module.exports = exports = async (options = DB) => {
 		log.info(`${MODULE_ID} creating new DB instance with options => `, options);
 
 		if (options.uri) {
-			const conn = Object.assign({ ssl: { rejectUnauthorized : false } }, pgConn.parse(options.uri));
+			const conn = Object.assign({ ssl: { rejectUnauthorized : false, sslmode: require } }, pgConn.parse(options.uri));
```

Comment:
**Issue:** `require` is a function (commonjs' [`require('module')`](https://nodejs.org/docs/latest/api/modules.html#modules_require_id)) here. This should be set as a string.

Reply:
Ooof, good catch 🤦🏻